### ISSUE TITLE
Request: Specify column by name rather than number

### ISSUE THREAD

**Issue by sjackman** [agordon/datamash#1]
With `--header-in`, it would be great to be able to specify the column by name rather than number.

```
❯❯❯ datamash -H -g3 sum 4 <expenses.tsv
GroupBy(Currency)	sum(Amount)
CDN	37.69
US	149.32
❯❯❯ datamash -H -g Currency sum Amount <expenses.tsv
datamash: invalid field value for grouping 'Currency'
❯❯❯ datamash -H -g3 sum Amount <expenses.tsv
datamash: invalid column 'Amount' for operation 'sum'
```

**Comment by sjackman** [comment]
Any plans to release 1.0.7 with this feature?

**Comment by sjackman** [comment]
One test of `make check` of 1.0.6.34-2362 fails. See below.

```
FAIL: tests/datamash-tests.pl (exit: 1)
=======================================

datamash-tests.pl: test e5: stderr mismatch, comparing e5.2 (expected) and e5.E 
*** e5.2        Fri Feb 13 14:31:50 2015
--- e5.E        Fri Feb 13 14:31:50 2015
***************
*** 1 ****
! datamash: invalid grouping parameter ''
--- 1 ----
! datamash: -H or --header-in must be used with named columns
```

**Comment by agordon** [comment]
Hello!

Hope to release version 1.0.7 very soon,
thanks for the bug report, I'll look into it.

-gordon

**Comment by sjackman** [comment]
I added a `--HEAD` option to the Homebrew formula. https://github.com/Homebrew/homebrew/pull/39736
Any plans for a 1.0.7 release?

**Comment by agordon** [comment]
Hello @sjackman,

I have a new tarball ready for pre-release testing, is there a way to check it on Mac OS X 10.10 and 10.8 (perhaps 10.11?) - I don't have access to such systems, and I'd prefer not to make an official 1.0.7 release before testing on those:
http://files.housegordon.org/datamash/src/datamash-1.0.6.54-f8ae.tar.gz

Thanks!

**Comment by sjackman** [comment]
Works for me on Mac OS 10.10.3. Thanks, @agordon.

**Comment by sjackman** [comment]
Fixed by 1.0.7.
